### PR TITLE
Convert @fluentui/react-radio stories to index.stories.tsx approach

### LINKED ISSUE #23581: Convert @fluentui/react-radio stories to index.stories.tsx approach

### PR DESCRIPTION
**Pull request by micahgodbolt**
Fixes #23581

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `185.285 kB`<br />`51.364 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `33.988 kB`<br />`11.108 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `29.813 kB`<br />`10.155 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |    `13.595 kB`<br />`5.42 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c38b5bd4e52a9542840e614ef07557eb8e692067'>c38b5bd4e52a9542840e614ef07557eb8e692067</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/Avatar-mount_base.html">1403</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/Avatar-mount.html">1389</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/Button-mount_base.html">1007</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/Button-mount.html">1036</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProvider-mount_base.html">2294</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProvider-mount.html">2323</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProviderWithTheme-mount_base.html">755</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProviderWithTheme-mount.html">720</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">716</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">796</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/MakeStyles-mount_base.html">2001</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23582/perf-test-react-components/MakeStyles-mount.html">2048</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-radio/src/stories/RadioGroup/RadioGroupDefault.stories.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Label } from '@fluentui/react-label';
 import { tokens } from '@fluentui/react-theme';
 import { useId } from '@fluentui/react-utilities';
-import { Radio, RadioGroup, RadioGroupProps } from '../index';
+import { Radio, RadioGroup, RadioGroupProps } from '@fluentui/react-radio';
```
Comment (khmakoto) [inline packages/react-components/react-radio/src/stories/RadioGroup/RadioGroupDefault.stories.tsx:5]: _nit:_ You could split these up into

```suggestion
import { Radio, RadioGroup } from '@fluentui/react-radio';
import type { RadioGroupProps } from '@fluentui/react-radio';
```

**Review by khmakoto — APPROVED** [review]

File: packages/react-components/react-radio/src/stories/RadioGroup/RadioGroupControlledValue.stories.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Label } from '@fluentui/react-label';
 import { tokens } from '@fluentui/react-theme';
 import { useId } from '@fluentui/react-utilities';
-import { Radio, RadioGroup } from '../index';
+import { Radio, RadioGroup } from '@fluentui/react-radio';
```
Comment (spmonahan) [inline packages/react-components/react-radio/src/stories/RadioGroup/RadioGroupControlledValue.stories.tsx:5]: Why are we making the change to use the package import?

**Pull request merged**